Formula/fossil.rb

```diff
@@ -5,6 +5,7 @@ class Fossil < Formula
   sha256 "4f135659ec9a3958a10eec98f79d4d3fc10edeae2605b4b38e0a58826800b490"
   license "BSD-2-Clause"
   head "https://www.fossil-scm.org/", using: :fossil
+  revision 1
```

Comment:
```suggestion
```
Let's skip this. We don't need to force everyone to reinstall this formula for shell completions.

Reply:
done.